### DISCUSSION TITLE
minimum configuration not working

### DISCUSSION THREAD

**Discussion by 4nti3r** [TypQxQ/KTC#31] [Configuring] [answered]
Hey Andrei,

first of all thanks for your work on this project. 

As I'm trying to get my mechanical toolchanger (based on a voron trident with an octopus-board) to work, I am getting errors - also when trying your example configurations. So I tried to generate a minimal setup on my own with two heads. my config-file is atached.

The actual code does not throw any errors so far :) But when I'm trying to add a pickup_gcode for my tool 
![image](https://github.com/user-attachments/assets/7dd9bb9c-f7a3-4b2f-9552-f45541b1e63a), I get the error:  "Option 'pickup_gcode' is not valid in section 'ktc_tool 0'".

It would be great if you (or someone else) could help me with this issue and give me a hint, what I missed.

[config.zip](https://github.com/user-attachments/files/16908827/config.zip)

Greetings,
4nti3r

**Comment by TypQxQ** [comment, marked as answer]
Hi!
I belive it's called "tool_select_gcode"
See here for example:
https://github.com/TypQxQ/KTC/blob/ed288bf0cebdd1d7cec572ee3e8f37c13e413878/config/example_config/simple/simple_example.cfg#L20

And here for documentation:
https://github.com/TypQxQ/KTC/blob/main/doc/configuration.md#----inheritable-options

Greetings,, Andrei

**Comment by 4nti3r** [comment]
Hey,
thanks for your help. It's not throwing the error again :)

My fault, I was using the config from KTC/config/TOOLS.cfg - which I now assume is an old version.

Thanks again :)

**Comment by TypQxQ** [comment]
Hmm, that was some old code I must have forgotten to remove..
Thank you.
